python/src/trezorlib/cli/cardano.py

```diff
@@ -59,14 +60,46 @@ def sign_tx(client, file, protocol_magic):
 
 
 @cli.command()
-@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-n", "--path-str", required=True, help=PATH_HELP)
 @click.option("-d", "--show-display", is_flag=True)
-@click.option("-p", "--protocol-magic", type=int, default=1)
+@click.option(
+    "-t", "--address-type", type=int, default=messages.CardanoAddressType.BASE
+)
+@click.option("-s", "--staking-key-path-str", type=str, default=None)
+@click.option("-h", "--staking-key-hash-str", type=str, default=None)
+@click.option("-b", "--block_index", type=int, default=None)
+@click.option("-x", "--tx_index", type=int, default=None)
+@click.option("-c", "--certificate_index", type=int, default=None)
+@click.option("-p", "--protocol-magic", type=int, default=0)
+@click.option("-N", "--network-id", type=int, default=0)
 @with_client
-def get_address(client, address, show_display, protocol_magic):
+def get_address(
+    client,
+    path_str,
+    address_type,
+    staking_key_path_str,
+    staking_key_hash_str,
+    block_index,
+    tx_index,
+    certificate_index,
+    protocol_magic,
+    network_id,
+    show_display,
+):
     """Get Cardano address."""
```

Comment:
please put a brief description of which parameters should match up here

Reply:
Added [here](https://github.com/trezor/trezor-firmware/pull/1112/commits/8c4a5c8a733d9d991405e5ee1c4f9d9bfe9895bf).